doc/bootc.rst

```diff
@@ -35,5 +38,26 @@ All general DNF options are accepted, see `Options` in :manpage:`dnf(8)` for det
 ``--pending-exit-77``
     If pending deploymewnt available, exit 77 (valid on the status subcommand).
 
+``--uninstall PKG``
```

Comment:
should also clarify this supports multiple pkgspecs if we want to support that, see below comment

Reply:
Might as well.